src/ckb-daemon/structures.h

```diff
@@ -247,7 +248,7 @@ typedef struct {
     // Color dithering in use
     char dither;
     // Flag to check, if large macros should be sent delayed
-    char delay;
+    uint delay;
```

Comment:
Yes, on two accounts, the other changes use this as a uint, and the syntax the GUI uses works with the these changes.